src/Microsoft.AspNet.WebUtilities/FileBufferingReadStream.cs

```diff
@@ -88,6 +101,13 @@ public override long Seek(long offset, SeekOrigin origin)
 
         private Stream CreateTempFile()
         {
+            if (_tempFileDirectory == null)
```

Comment:
Will this ever evaluate to true with the [NotNull] attribute in the ctor on tempFileDirectory?